manifests/e/Enterbrain/RGSS-RTPStandard/1.0.4.0/Enterbrain.RGSS-RTPStandard.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Enterbrain.RGSS-RTPStandard
+PackageVersion: 1.0.4.0
+InstallerSwitches:
+  Silent: /s

```

Comment:
```suggestion
```
These aren't needed by the way because winget-cli automatically adds the switches when installing it from WinGet.